change/@microsoft-teams-js-4885d054-ef67-46ee-ba6a-df10fdf1a3eb.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "update interfaces to refer to updated app context interface",
```

Comment:
Can you please modify this so it shows up consistently in our change log and adds a little more detail about the change (guidelines are in [CONTRIBUTING.md](https://github.com/OfficeDev/microsoft-teams-library-js/blob/main/CONTRIBUTING.md#change-log-using-beachball))? 

E.g. (please make sure to use backticks around `Context` and `app.Context`),

"Updated reference documentation for global deprecated `Context` interface. Deprecated `Context` properties now link to respective mapped property in `app.Context` interface."

Reply:
did you want me to look in the CONTRIBUTING.md or did you want me to write what you just wrote in quotes?